### ISSUE TITLE
No resolvable combination of libpq=16 openssl=3.2 libxml2=2.11

### ISSUE THREAD

**Issue by akrherz** [conda-forge/postgresql-feedstock#194]
### Comment:

With the pain involved this week with #191 and the builds that got marked broken as a result, we are left with an unresolvable combination of this:

```
$ mamba create -n dev libpq=16 openssl=3.2 libxml2=2.11
Could not solve for environment specs
The following packages are incompatible
├─ libpq 16**  is installable with the potential options
│  ├─ libpq 16.0 would require
│  │  └─ openssl >=3.1.3,<3.2.0a0 , which can be installed;
│  ├─ libpq 16.1 would require
│  │  └─ libxml2 >=2.12.2,<2.13.0a0 , which can be installed;
│  └─ libpq 16.1 would require
│     └─ openssl >=3.1.4,<3.2.0a0 , which can be installed;
├─ libxml2 2.11**  is not installable because it conflicts with any installable versions previously reported;
└─ openssl 3.2**  is not installable because it conflicts with any installable versions previously reported.
```

This is not allowing gdal=3.8.1 to install, see conda-forge/gdal-feedstock#867

It may be helpful to make an intermediate build against `libxml2 2.11`, but can certainly understand if there isn't an appetite here to go messing with postgresql builds for a while :/

**Comment by akrherz** [comment]
should have been fixed by #197, but will verify first once builds propagate CDN

**Comment by akrherz** [comment]
appears resolved
```
$ mamba create -n dev libpq=16 openssl=3.2 libxml2=2.11
 + openssl                 3.2.0  hd590300_1   conda-forge     Cached
  + libxml2                2.11.6  h232c23b_0   conda-forge     Cached
  + libpq                    16.1  h33b98f1_7   conda-forge        2MB
```
